### DISCUSSION TITLE
Neue Dateien im Medienbrowser kennzeichnen

### DISCUSSION THREAD

**Discussion by BillyTheKing** [lotta-schule/web#230] [Ideen]
**Überlegung**: neu hochgeladene Dateien und neu erstellte Ordner kennzeichnen

**Idee**: mit einem Punkt wie bei neuen Beiträgen in der Beitragsvorschau

Zur Diskussion im Team

**Comment by wie0815** [comment]
Hatten wir schon öfter auf dem Schirm. Ich würde es hervorragend finden! Gerne als ToDo ... wenn @ptitmouton teilbare Ordner ermöglicht, wäre das ein Feature mit echtem Mehrwert.

**Comment by techotline** [comment]
@wie0815 Für 'Teilbare Ordner' existiert ein Ticket (#197). Möglicherweise nicht Thema von Marker für neue Dateien.
Sozusagen habe ich die Idee aufgegriffen und meine Gedanken in einem Ticket verpackt, jedoch ist Eure Vorstellung / Ziel für das Dev von teilbare Ordner eine andere, oder? (Diskussion: das geht so nicht, wie Du Dir das vorstellst ...)

**Comment by ptitmouton** [comment]
Punkt eins vor Allem anderen ist festzulegen, was wir damit erreichen wollen. 

Daraus können wir dann ableiten, was genau "neu" in diesem Zusammenhang bedeuten soll.

Erst dann kann technischer Aufwand eingeschätzt werden und ob es überhaupt irgendeine Verbindung zu anderen Plänen geben könnte.

@wie0815 Vielleicht kannst du erklären warum du das schon öfter auf dem Schirm hattest.

**Comment by ptitmouton** [comment]
> @wie0815 Vielleicht kannst du erklären warum du das schon öfter auf dem Schirm hattest.

@wie0815 kannst du bitte erläutern was ‚neu‘ in dem Fall für dich bedeutet bzw. was es bedeuten müsste um hilfreich zu sein?

**Comment by wie0815** [comment]
Was ist im Dateibrowser neu? ... verstehe ich so, dass Person A eine Datei hochlädt und Person B anhand einer Markierung erkennt, dass hier etwas Neues im Ordner ist. Mach natürlich nur in Ordnern Sinn, auf die mehrere Personen Zugriff haben. #"teilbare Ordner".

**Comment by ptitmouton** [comment]
@wie0815 teilbare Ordner gibt es noch nicht.
Heißt das es geht überhaupt nicht um eine Erweiterung der aktuellen Funktionalität?
wenn ja, hast du das auch so gemeint @BillyTheKing ?
In dem Fall würde ich das Ticket zumachen. Wir können ja schlecht über Veränderungen zu einer Sache diskutieren, von der wir gar nicht wissen was sie ist. 

Vermutlich werden sich in dem Zusammenhang dieselben Fragen wieder stellen, hätte eine Antwort interessant gefunden.

Schließen?

**Comment by techotline** [comment]
Meiner Meinung nach ist dieses Ticket die Annotation zu "Teilbare Ordner".

... die natürlich noch nicht existieren, weil geklärt werden muss, welchen Zweck diese erfüllen. #197

Warum Dateien kennzeichnen, die man selbst zuvor in den privaten Medienbrowser abgelegt hat, wenn nur eine Person (in dem Fall der User) darauf zugreifen kann.

**Comment by ptitmouton** [comment]
Ja warum, das habe ich hier schon zwei mal gefragt ohne eine Antwort zu bekommen. Wenn das Feature keiner will und auch keiner etwas konkretes beitragen kann, dann mache ich das Ticket zu. Das ist doch sonst alles nur Gewäsch.

@BillyTheKing ich mach das Ticket zu, es scheint keiner zu wollen. Solltest du das anders sehen mach es wieder auf und erläutere was gemeint war.
